src/services/keycloak/auth.ts

```diff
@@ -22,6 +22,8 @@ export type IKeycloakUserInfo = {
   sub: string;
 };
 
+const MAX_VALIDITY_TIME = 3600;
```

Comment:
It's a good practice to make it clear time unit with const or var name, like the following
```suggestion
const MAX_VALIDITY_TIME_MIN = 3600;
```
^ I'm not sure if it's mins or not